ui/src/views/image/RegisterOrUploadTemplate.vue

```diff
@@ -800,6 +812,8 @@ export default {
             params[key] = input
           } else if (key === 'hypervisor') {
             params[key] = this.hyperVisor.opts[input].name
+          } else if (key === 'deployasis') {
+            params[key] = input
```

Comment:
Shouldn't this be handled by default at 841-842? Else, is it better to club 809, 811 and this

Reply:
Thanks, fixed